packages/blade/src/components/Checkbox/__tests__/__snapshots__/Checkbox.web.test.tsx.snap

```diff
@@ -82,7 +82,7 @@ exports[`<Checkbox /> should render checkbox with label 1`] = `
   font-style: normal;
   -webkit-text-decoration-line: none;
   text-decoration-line: none;
-  line-height: 1.25rem;
+  line-height: 1rem;
```

Comment:
Because this test uses Link component 🤦  Nothing to worry about

```jsx
it('should render checkbox with label', () => {
    const label = (
      <>
        I accept <Link href="#">Terms and Conditions</Link>
      </>
    );
```